docs/man_pages/lib-management/kendoui-install.md

```diff
@@ -0,0 +1,51 @@
+kendoui install
+==========
+
+Usage | Synopsis
+------|-------
+List all versions | `$ appbuilder kendoui install [--verified] [--latest]`
```

Comment:
the main reason is that the server can return different results and this should not be automated by using index. For example I may create automated test that adds version 2, after 1 month, this version will not be the one I expect, because the server returns different results. 
I prefer to keep using "flags" like --latest, --core, etc. so the user will not be able to automate some state and after that wonder "why it is not working as before"